inc/comments-template.php

```diff
@@ -11,12 +11,38 @@
 $talk_url = get_option( 'coral_talk_base_url' );
 $static_url = get_option( 'coral_talk_static_url', $talk_url );
 $talk_container_classes = get_option( 'coral_talk_container_classes' );
+$talk_version = get_option( 'coral_talk_version' );
+
 $div_id = 'coral_talk_' . absint( rand() );
-if ( ! empty( $talk_url ) ) : ?>
+
+if ( empty( $talk_url) ):
+	exit();
+endif;
+
+if ( $talk_version == "4" ) : ?>
```

Comment:
For fallback, you should instead do something like `if ( $talk_version ~= "5" ) : ?>` instead, as those who have the plugin installed already should still load the correct template.